sdk/storage/storage-blob/package.json

```diff
@@ -102,9 +102,9 @@
     "@azure/core-http": "^1.1.4",
     "@azure/core-lro": "^1.0.2",
     "@azure/core-paging": "^1.1.1",
-    "@azure/core-tracing": "1.0.0-preview.8",
+    "@azure/core-tracing": "1.0.0-preview.9",
     "@azure/logger": "^1.0.0",
-    "@opentelemetry/api": "^0.6.1",
+    "@opentelemetry/api": "^0.10.2",
```

Comment:
Besides "change compile target to es5", is there any other changes involved and could influence end user?

Reply:
Looks this is the only change that affects us
![image](https://user-images.githubusercontent.com/7583839/89330070-912d3280-d644-11ea-919e-d40fc0b6f5d8.png)

Reply:
@jeremymeng: You want to look at all breaking changes between `0.6.1` and `0.10.2`, not just the changes in the `0.10.2` release.  I personally do not know if or how any of these breaking changes might impact our packages.

Reply:
Fortunately we use very little of their public surface area and active development has been in other areas

Reply:
oh that's right I was only looking at last release. But Jeff knows more!

Reply:
I went through all the release notes from `0.6.1` -- there were breaking changes in metrics (which we don't use yet), but not tracing.